src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -345,7 +345,9 @@ public SDKRestClient admin() {
          * @return An instance of a cluster admin client.
          */
         public SDKClusterAdminClient cluster() {
-            return new SDKClusterAdminClient(restHighLevelClient.cluster());
```

Comment:
I'm not sure I like these changes.  Having a final field set in the constructor allows for more compiler optimizations than a nonfinal field with a setter.  Was this a spotbugs issue?